### ISSUE TITLE
missing dependency on ghc-heap

### ISSUE THREAD

**Issue by Profpatsch** [def-/ghc-datasize#13]
with version 2.5 I get:

```
[1 of 1] Compiling GHC.DataSize     ( src/GHC/DataSize.hs, dist/build/GHC/DataSize.o, dist/build/GHC/DataSize.dyn_o )

src/GHC/DataSize.hs:23:1: error:
    Could not load module ‘GHC.Exts.Heap’
    It is a member of the hidden package ‘ghc-heap-8.10.7’.
    Perhaps you need to add ‘ghc-heap’ to the build-depends in your .cabal file.
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
23 | import GHC.Exts.Heap hiding (size)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/GHC/DataSize.hs:24:1: error:
    Could not load module ‘GHC.Exts.Heap.Constants’
    It is a member of the hidden package ‘ghc-heap-8.10.7’.
    Perhaps you need to add ‘ghc-heap’ to the build-depends in your .cabal file.
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
24 | import GHC.Exts.Heap.Constants (wORD_SIZE)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

```
> ghc --version
The Glorious Glasgow Haskell Compilation System, version 8.10.7
```

**Comment by Profpatsch** [comment]
2.4 compiles just fine.

**Comment by Profpatsch** [comment]
The hackage matrix builder shows the same error: https://matrix.haskell.org/#/package/ghc-datasize/0.2.5/ghc-8.10.1@1634405803

**Comment by def-** [comment]
Thanks, I hope it's fixed in 0.2.6
